### DISCUSSION TITLE
Unusable on Raspberry Pi 3B+

### DISCUSSION THREAD

**Discussion by alazyworkaholic** [wezterm/wezterm#5211] [Q&A]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

i3

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I've just installed wezterm on my Raspberry Pi 3B+ by adding https://apt.fury.io/wez to my sources.
Running Raspbian OS (Bookworm) ; ARM64 ; Linux 6.6.21 ; glx 1.4 ; Open GL 2.1 ; Open GL ES 2.0 ; OpenGL Shading Language 1.20 ; OpenGL ES GLSL ES 1.0.16 ; Mesa 23.2.1-1~bpo12+rpt3 ; only notable system change from clean os install is installing regolith-desktop and using i3 session.

Performance is atrocious. ~500ms lag from key press to display. top shows wezterm-gui's CPU usage at ~200% from slowly typing the alphabet.
The Pi 3B+ is weak, but something must be wrong. The default LX Terminal is snappy and I expected wezterm to offer even better performance.
Maybe wezterm can't use the RPi 3B+'s GPU, like zutty? Am I missing some crucial configuration?

**Comment by wez** [comment]
This looks like a (poorly specified) bug report in the guise of a discussion thread. You won't get any usable responses here.
Consider opening an issue and including actionable information in it.
However, in the interest of transparency: I have no time or desire to make wezterm run well on such an under powered device, so if this is anything beyond a simple misconfiguration you're better off using something else.

**Comment by searleser97** [comment]
raspberry PIs now have become more powerful, could you provide pointers, on how could we make it work there ? Thanks

**Reply by searleser97** [reply]
just figured that running it with `wezterm --config 'front_end="Software"'` fixes the issue :)
